src/ParkDetails/ParkDetails.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./ParkDetails.css";
+
+function ParkDetails({ foundPark }) {
+  if (!foundPark) {
```

Comment:
is foundPark just the selected park?

Reply:
Yes. Its currently set to be overwritten by its useState every time a new park is chosen.